src/main/java/com/wavefront/opentracing/WavefrontTracer.java

```diff
@@ -311,6 +311,7 @@ void reportWavefrontGeneratedData(WavefrontSpan span) {
     if (span.isError()) {
       Map<String, String> errorPointTags = new HashMap<>(pointTags);
       errorPointTags.put("error", "true");
+      errorPointTags.put(HTTP_STATUS.getKey(), spanTags.get(HTTP_STATUS.getKey()).iterator().next());
```

Comment:
should also handle case where span doesn't have http status tag. i think we can exclude it if that's the case, as opposed to setting the tag value to a default.